tools/generate-helpers/pg-table-bindings/store_test.go.tpl

```diff
@@ -219,7 +219,11 @@ func (s *{{$namePrefix}}StoreSuite) getTestData(access storage.Access) (*{{.Type
 		},
 		withAccess: {
 			context:                sac.WithGlobalAccessScopeChecker(context.Background(),
+				{{- if .Obj.IsNamespaceScope }}
```

Comment:
Doesn't this always evaluate to `sac.AllowFixedScopes`?

Reply:
In the current state of the PR, it indeed does. At some point, the template was migrated to the `AllowFixedXXXScopes` variants, and it made sense to differentiate.
It could be simplified here, but would have to be re-adjusted in #6924 .

Reply:
Alright I see, thanks.